src/fidesops/util/api_router.py

```diff
@@ -0,0 +1,35 @@
+from typing import Any, Callable
+
+from fastapi import APIRouter as FastAPIRouter
+from fastapi.types import DecoratedCallable
+
+
+class APIRouter(FastAPIRouter):
+    """
+    Taken from: https://github.com/tiangolo/fastapi/issues/2060#issuecomment-834868906
+    """
+
+    def api_route(
+        self, path: str, *, include_in_schema: bool = True, **kwargs: Any
+    ) -> Callable[[DecoratedCallable], DecoratedCallable]:
+        """
+        Updated api_route function that automatically configures routes to have 2 versions.
+        One without a trailing slash and another with it.
+        """
+        if path.endswith("/"):
+            path = path[:-1]
+
+        add_path = super().api_route(
+            path, include_in_schema=include_in_schema, **kwargs
+        )
+
+        alternate_path = path + "/"
+        add_alternate_path = super().api_route(
+            alternate_path, include_in_schema=False, **kwargs
+        )
+
+        def decorator(func: DecoratedCallable) -> DecoratedCallable:
+            add_alternate_path(func)
+            return add_path(func)
+
+        return decorator
```

Comment:
Interesting solution, to handle trailing slashes. Main thing, we talked about a test for this one, otherwise, it's looking good

Reply:
Would echo what Dawn has said there. It's nice for the server to behave the same regardless of a trailing slash. Django for instance solves this with regex in the URL def where you can say something like:
```
    re_path(r'^api/v1/some-endpoint/?', comments),
```
where the last `?` makes the preceding `/` optional. Since we're effectively doing this automatically now, it would be good to also document that somewhere in the API docs.